pages/index.html

```diff
@@ -1,12 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="../styles/index.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css">
+    <title>Document</title>
 </head>
+
 <body>
-  
+    <section class="showcase">
+        <header>
+            <h1 class="logo">personal Portfolio website</h1>
+            <div class="toggle"></div>
+        </header>
+        <video src="../video/seattle1.mp4" muted loop autoplay>
+            </video>
+        <div class="overlay"></div>
+        <div class="text">
+            <h2>Vida Maleki </h2>
+            <h3>My Personal Portfolio</h3>
+            <p>This website(portfolio) created for unit3 project in Ada Developer Academy. The porpose is to test our knowledge about HTML & CSS.</p>
+            <a href="portfolio.html">Portfolio</a>
+            <a href="about.html">About</a>
+        </div>
+        <ul class="social">
+
+            <li>
+                <a href="https://twitter.com/vida_maleki"><img src="https://i.ibb.co/Wnxq2Nq/twitter.png" alt="face book logo"></a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/vida-ghorbannezhad-maleki-4082a4197/"><i class="fab fa-linkedin"style='font-size:40px; color:rgb(8, 8, 8)'></i></a>
+            </li>
+            <li>
+                <a href="https://github.com/VidaMaleki"><i class="fab fa-github" style='font-size:40px; color:rgb(8, 8, 8)'></i></a>
+            </li>
+            <li>
+                <footer>
+                    <p>&copy;VidaMaleki</p>
+                </footer>
+            </li>
+        </ul>
```

Comment:
This looks nice! To make it a bit more semantic, we probably want the `footer` wrapping the `ul`, not the other way around.